thevoid/connection.cpp

```diff
@@ -542,12 +551,17 @@ void connection<T>::print_access_log()
 		m_access_status,
 		m_access_received,
 		m_access_sent,
-		delta);
+		delta,
+		m_receive_time.count(),
+		m_send_time.count());
```

Comment:
You cannot use %lld to printf chrono's underlying type, because the Standart specifies only the minimum amount of bits in these types, but not exactly types, therefore you cannot be sure the type is fit by this printf's option.